### DISCUSSION TITLE
Lesson 11

### DISCUSSION THREAD

**Discussion by itsmohammadh** [Cyfrin/foundry-full-course-cu#1015] [General]
hi i cant recieve NFT in metamask why?

i see this error in metamask when i import contract👇

NFT can’t be added as the ownership details do not match. Make sure you have entered correct information.

how can i fix that?

**Comment by 0xusmanf** [comment]
Make sure you are entering the correct `Token ID` of the NFT that you want to import and that you actually own it.

You can make sure about the `Token ID` by searching your address on etherscan and then click on `NFT Transfers` and then click on the Transaction of the corresponding NFT that you want to import. 

You will see the `Token ID` like below.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/82a55b1f-2c82-4ef3-80e3-9a8aa269d819)

**Reply by theirrationalone** [reply]
Heyyy @usmanfarooq91 !, Let me add one more crucial step which can be a silly omission 😅

@itsmohammadh !, Please be sure to use `FCN` Address along with `Token ID` to import your NFT.

**Reply by itsmohammadh** [reply]
Thank you @usmanfarooq91 🙏
